app/static/js/slcNewProposal.js

```diff
@@ -413,11 +413,22 @@ function enableSyllabusUploadFile() {
 }
 
 const textareas = $(".textarea");
-const slcQuestionCharCount = $(".slcQuestionCharCount");
+const slcQuestionWordCount = $(".slcQuestionWordCount")
```

Comment:
We don't need the `.slcQuestionWordCount` class at all, actually. We can still get all of those elements with this selector: $(".slcQuestionWordCounter span")

Reply:
resolved